src/util/math.ts

```diff
@@ -0,0 +1,13 @@
+// euclidean distance between two points
+export const dist = (x1 = 0, y1 = 0, x2 = 0, y2 = 0): number =>
```

Comment:
Just out of curiosity, why specify default values for this file's function's arguments?

Reply:
It's an easy/terse way to have typescript infer that they are number types, and doubles as a safe guard. At one point in my experimentation/iteration I was also using it like `dist(x,y)` to get the distance of a point to the origin.